db/migrations/20180829161115_add_fx_newsletter_column.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+exports.up = function(knex, Promise) {
+  return knex.schema.table("subscribers", table => {
+    table.boolean("fx_newsletter").defaultTo(false);
+  });
+};
+
+exports.down = function(knex, Promise) {

```

Comment:
Not sure what's up with the odd little unused `Promise` argument in these, but I'll assume it's just a **knex** oddity.